R/benchmark.R

```diff
@@ -52,6 +52,10 @@
 #' * `cpu_count`: To restrict the number of threads available for computation,
 #' specify an integer `cpu_count`. This sets the R `Ncpus` option, which many
 #' packages follow, and also caps the `arrow` threadpool size.
+#' * `mem_alloc`: The memory allocator to be tested (one of: "jemalloc",
+#' "mimalloc", "system")
+#' * `drop_caches`: Attempt to drop the disk cache before each iteration.
+#' Currently only works on linux and defaults to `FALSE`.
```

Comment:
This is technically not accurate since it _doesn't_ (yet) drop them before each iteration right? I do think we should allow both (see my larger comment + also Austin's). So this is probably fine to leave as is, but if we decide _not_ to do that, we should come back and fix this

Reply:
Oops, yeah, this is an artifact of refactoring. I'll make sure it reflects the code after updates